Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml

```diff
@@ -1909,3 +1909,28 @@
     - CottonBoll
   - type: Extractable
     grindableSolutionName: food
+
+- type: entity
+  name: coffee
+  parent: FoodProduceBase
+  id: FoodCoffee
+  description: Red berries encasing coffee beans.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - bitter
+  - type: Food
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 6
+        reagents:
+        - ReagentId: CoffeeGrounds
+          Quantity: 5
```

Comment:
Here's a suggestion for you, might be fun.

Instead of grinding the seeds directly into coffee grounds, why not make something similar to the cocoa plant?  Either eat the fruit or use it in your hands to get raw coffee beans.

Cook the coffee beans on a grill until they're roasted (refer to steak to see how this is done).

Then, grind the coffee beans in a juicer.

Reply:
Of course, we had a discussion about that on discord. I wanted to keep this as simple as possible. I think there is room for an netire game loop, a space barista, and if I would ever get around to this, once I get experience it could be revisited.


Additional steps with beans, possibly coffee robusta mutation, an espresso machine, more options like white, americano or latte, etc.

For now, I just wanted to plug a hole that we noticed each time we ran a food truck - cant offer coffee because we have no dispenser and cant grow it!